content/SCALETutorials/Instances/_index.md

```diff
@@ -19,554 +19,657 @@ keywords:
 
 {{< include file="/static/includes/25.04Virtualization.md" >}}
 
-**Instances** allow users to set up containers or virtual machines (VMs) to run alongside TrueNAS.
+**Instances** allow users to configure Incus-based containers or VMs in TrueNAS.
 
-A *Linux container* is a lightweight, isolated environment that shares the host system kernel while maintaining its own filesystem, processes, and network settings.
-Containers start quickly, use fewer system resources than VMs, and scale efficiently, making them ideal for deploying and managing applications with minimal overhead.
+*Linux containers*, powered by LXC, offer a lightweight, isolated environment that shares the host system kernel while maintaining its own file system, processes, and network settings.
```

Comment:
I didn't catch this the first time but we don't need to use the emphasis for defining a term here by putting Linux containers in italics as it is plain enough to understand with the explanation following it. It is not a big deal but not really needed here.
```suggestion
Linux containers, powered by LXC, offer a lightweight, isolated environment that shares the host system kernel while maintaining its own file system, processes, and network settings.
```